cf/cf-jobs.yml

```diff
@@ -283,10 +299,9 @@ jobs:
         zone: z1
       route_registrar:
         routes: (( merge || meta.uaa_routes ))
-      router:
-        servers:
-          z1: (( jobs.router_z1.networks.cf1.static_ips ))
-          z2: (( jobs.router_z2.networks.cf2.static_ips ))
+      uaa:
+        proxy:
```

Comment:
@LinuxBozo Do you know what the proxy is for? Those are not the end IPs for the UAA